test/IntegrationTests/NLogBridgeTests.cs

```diff
@@ -0,0 +1,189 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Text.RegularExpressions;
+using Google.Protobuf;
+using IntegrationTests.Helpers;
+using OpenTelemetry.Proto.Logs.V1;
+using Xunit.Abstractions;
+
+namespace IntegrationTests;
+
+public class NLogBridgeTests : TestHelper
+{
+    public NLogBridgeTests(ITestOutputHelper output)
+        : base("NLogBridge", output)
+    {
+    }
+
+    [Theory]
+    [Trait("Category", "EndToEnd")]
+    [MemberData(nameof(LibraryVersion.NLog), MemberType = typeof(LibraryVersion))]
+    public void SubmitLogs_ThroughNLogBridge_WhenNLogIsUsedDirectlyForLogging(string packageVersion)
+    {
+        using var collector = new MockLogsCollector(Output);
+        SetExporter(collector);
+
+        // Logged in scope of an activity
+        collector.Expect(
+            logRecord =>
+            VerifyBody(logRecord, "{0}, {1} at {2:t}!") &&
+            VerifyTraceContext(logRecord) &&
+            logRecord is { SeverityText: "Info", SeverityNumber: SeverityNumber.Info } &&
+            VerifyAttributes(logRecord) &&
+            logRecord.Attributes.Count == 4,
```

Comment:
Using NLog directly has 4 log record attributes vs 3 log record attributes when using ILogger with the NLog provider. What's different in the sets of attribute?

Reply:
The custom attribute `"test_key": "test_value"` is only present when using the NLog Bridge directly but gets filtered out when using the ILogger Bridge. 

This suggests:
NLog Bridge: Preserves all NLog-specific properties and custom attributes
ILogger Bridge: Filters out some NLog-specific attributes, keeping only the structured logging parameters

This makes sense because:
Direct NLog usage has full access to NLog's property system

ILogger usage goes through Microsoft's logging abstraction, which may not preserve all NLog-specific metadata
The test correctly validates that the two different logging paths (direct NLog vs ILogger provider) produce different attribute sets, which is expected behavior